src/controller/python/chip/clusters/Attribute.py

```diff
@@ -65,17 +78,92 @@ def __hash__(self):
         return str(self).__hash__()
 
 
+@dataclass
+class EventPath:
+    EndpointId: int = None
+    ClusterId: int = None
+    EventId: int = None
+
+    def __init__(self, EndpointId: int = None, Cluster=None, Event=None, ClusterId=None, EventId=None):
+        self.EndpointId = EndpointId
+        if Cluster is not None:
+            # Wildcard read for a specific cluster
+            if (Event is not None) or (ClusterId is not None) or (EventId is not None):
+                raise Warning(
+                    "Event, ClusterId and AttributeId is ignored when Cluster is specified")
+            self.EventId = Event.id
+            return
+        if Event is not None:
+            if (ClusterId is not None) or (EventId is not None):
```

Comment:
I just wondering if you rely on their [truthy](https://docs.python.org/3/library/stdtypes.html#truth-value-testing) values:
```suggestion
        if Cluster:
            # Wildcard read for a specific cluster
            if Event or ClusterId or EventId:
                raise Warning(
                    "Event, ClusterId and AttributeId is ignored when Cluster is specified")
            self.EventId = Event.id
            return
        if Event:
            if ClusterId or EventId :
```

Reply:
EventId could be 0, so we would not this way, thanks for suggestion.